core/domain/html_cleaner.py

```diff
@@ -153,37 +153,25 @@ def escape_html(html_data):
         html_data: str. HTML string to be escaped.
 
     Returns:
-        str. Escaped HTML string
+        str. Escaped HTML string.
     """
-    replace_list = [
-        ('&', '&amp;'),
-        ('"', '&quot;'),
-        ('\'', '&#39;'),
-        ('<', '&lt;'),
-        ('>', '&gt;')
-    ]
+    replace_list = feconf.REPLACE_LIST
```

Comment:
The constant does not have to go in feconf. If it's only used within this file, keep it at the top of this file.

Reply:
(Also, you don't need to redeclare it here as a local variable -- just use it directly. Ditto below.)

Reply:
Done!